src/javascripts/components/navbar.js

```diff
@@ -0,0 +1,23 @@
+const navbar = () => {
+  const domString = `<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="container-fluid">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <div class="navbar-nav">
+        <a class="nav-link active" id="nav-menu" href="#">Menu<span class="sr-only">(current)</span></a>
```

Comment:
nit: I would update this line. It currently is indicating that the user is on the Menu Page ever when they are not.
```suggestion
        <a class="nav-link active" id="nav-menu" href="#">Menu</a>
```